test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml

```diff
@@ -19,7 +19,7 @@ spec:
           image: quay.io/k8scsi/csi-provisioner:v0.2.0
```

Comment:
is this correct?

Reply:
yea we haven't updated it yet, I will bump all the versions when we cut a beta of the GCE PD CSI Driver